crates/env/Cargo.toml

```diff
@@ -1,58 +1,60 @@
 [package]
 name = "ink_env"
-version = "4.2.0"
-authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
-edition = "2021"
-rust-version = "1.68"
 
-license = "Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/paritytech/ink"
-documentation = "https://docs.rs/ink_env/"
-homepage = "https://www.parity.io/"
+edition.workspace = true
+categories.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
+authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
 description = "[ink!] Low-level interface for interacting with the smart contract Wasm executor."
-keywords = ["wasm", "parity", "webassembly", "blockchain", "edsl"]
-categories = ["no-std", "embedded"]
+documentation = "https://docs.rs/ink_env/"
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
+readme = "README.md"
+rust-version = "1.68"
```

Comment:
Is that possible to use `workspace` for `rust-version` too?

Reply:
it is possible, but I wouldn't know what to put: there are two crates that actually define different versions, the others do not define; I believe that it should be 1.69, but this PR wasn't aiming at changing behaviour

Reply:
Oh, we do `rust-version` only for two crates. Then it is okay to keep it this way=)